### PR TITLE
Loading Screen

### LINKED ISSUE #26: Add Splash Screen While Site Is Loading
As @michaelfromyeg said in #24:

> Any thoughts on a splash screen while things are loading? When I first load (on desktop), seeing the solid brown background (before the background loads), then the images load in separately (first the sign post, then the background) still stings a bit. I know we're looking a number of other improvements as well, but I think at least having a design at least prepped for a splash screen could be useful.

### PR DESCRIPTION
**Pull request by rmehri01**
Closes #26 

## :construction_worker: Changes

- Moved old index to `Main.vue`
- Added a loading screen with fade in/out
- Made navbar consistent with faq/sponsor flags

## :flashlight: Testing Instructions

- Check out the amplify link!

### CODE REVIEW

File: components/Loading.vue [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+  z-index: 1000000;
+
+  .fire {
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -25%);
+    height: 20vw;
+    width: 20vw;
+    //background-color:$red;
```
Comment (kozr) [inline components/Loading.vue:44]: do we still need these comments?

File: components/Main.vue [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+$body-font: "Source Sans Pro", sans-serif;
+
+#main-page {
+  background: linear-gradient(180deg, #4A414C 8.34%, #061A2C 9.19%, #586971 12%, #172C3E 15%, #172C3E 22%, #67757B 26.66%, #B5B5B5 30.2%, #453F4B 34.08%);
+  font-family: $body-font;
+}
+
+@include until($tablet) {
+  #main-page {
+    background: linear-gradient(180deg, #4A414C 10%, #061A2C 12%, #586971 14%, #223040 18%, #223040 21%, #67757B 30%, #B5B5B5 30.2%, #453F4B 34.08%);
```
Comment (kozr) [inline components/Main.vue:90]: Are these values arbitrary?

Comment (rmehri01) [reply components/Main.vue:90]: Yeah I basically just adjusted them to work, I'm not really sure how to do it in a way that works for all screen sizes.

File: components/Loading.vue (hunk 1 above)
Comment (rmehri01) [reply components/Loading.vue:44]: I could remove them if you want, it was just from codepen LOL

File: pages/index.vue [hunk 3]
```diff
@@ -1,93 +1,43 @@
...
-        text: data.IntroText,
-        introButtonEnabled: data.IntroButtonEnabled,
-        introLink: data.IntroButtonLink,
-        signUpText: data.SignUpText
-      },
-      FAQs: FaqQuestions,
-      faqFlag: FaqQuestions.length !== 0 && data.faqFlag,
-      ...data.featureFlags
-    }
+    'async-component': AsyncComponent
```
Comment (kozr) [inline pages/index.vue:26]: Can we rename this to something similar to loading-screen? or better yet follow our current naming scheme and name it LoadingScreen or LoadPage?

**Review by kozr — APPROVED** [review]
Good work! It looks amazing. Let's have someone open it during our meeting to test the loading time.

File: components/Main.vue (hunk 2 above)
Comment (rmehri01) [reply components/Main.vue:90]: This isn't really related to this pr so I'll leave it for now.

**Pull request merged**